### PR TITLE
test omero53 in travis

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
enable travis test, 

--depends-on https://github.com/openmicroscopy/openmicroscopy/pull/5241
--depends-on https://github.com/openmicroscopy/openmicroscopy/pull/5243

### CODE REVIEW

**Comment by aleksandra-tarkowska** [comment]
it passes on:
- m52 https://10.0.51.135:8443/job/MAPR-test/
- m53 https://10.0.51.150:8443/job/MAPR-test/
- develop - travis

**Comment by aleksandra-tarkowska** [comment]
--exclude

**Comment by aleksandra-tarkowska** [comment]
restated travis https://travis-ci.org/ome/omero-mapr/builds/223456440 with the most recent server build https://travis-ci.org/ome/omero-mapr/builds/223456440#L805

**Comment by aleksandra-tarkowska** [comment]
mergable with https://github.com/openmicroscopy/openmicroscopy/pull/5243

**Comment by joshmoore** [comment]
Looks good. What else needs to happen before releasing `2.0`, @aleksandra-tarkowska ?

**Comment by aleksandra-tarkowska** [comment]
2 other prs has to be merged

**Comment by joshmoore** [comment]
Understood. :+1: for merging once the other PRs are in (unless you need to have this merged beforehand)

File: .travis.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ before_script:
 
 script:
   - echo 'Tested in CI'
```
Comment (joshmoore) [inline .travis.yml]: This can be moved. (Seen while working on https://github.com/ome/rOMERO-gateway/pull/14#issuecomment-299179708)

Comment (aleksandra-tarkowska) [inline .travis.yml]: ci is locked, no more changes at the moment

**Comment by aleksandra-tarkowska** [comment]
--exclude

**Comment by joshmoore** [comment]
Migrating to https://github.com/ome/omero-mapr/pull/23

**Pull request closed without merging**